service/tbc/tbc_test.go

```diff
@@ -69,6 +69,12 @@ func TestBtcBlockHeadersByHeight(t *testing.T) {
 	defer cancel()
 
 	bitcoindContainer, mappedPeerPort := createBitcoindWithInitialBlocks(ctx, t, 100, "")
+	defer func() {
+		if err := bitcoindContainer.Terminate(ctx); err != nil {
+			panic(err)
```

Comment:
If this panics, will it cause other contains to not be terminated?
If so, I think just logging the error would be better because it will allow all containers to be terminated

Reply:
it wouldn't cause other containers to not be terminated; testcontainers starts a "reaper" container, which is responsible for cleanup after the program completes.  The reaper would handle this.